README.md

```diff
@@ -97,15 +112,190 @@ A quick look at the top-level files and directories you'll see in a Gatsby proje
+  return <button {...props} />
+}
+
+export default Button
+```
+
+And, that's it! Now you have a working button that you can use anywhere on your project. FastStore, however, brings a handy library called `@vtex/store-ui` with built-in components to help you speed up your development. To use it, just change `Button.tsx` to:
+
+```tsx
+import React from 'react'
+import { Button as UIButton } from '@vtex/store-ui'
+import type { ButtonProps } from '@vtex/store-ui'
+
+interface Props extends ButtonProps {}
+
+function Button (props: Props) {
+  return <UIButton {...props} />
+}
+
+export default Button
+```
+
+Now, your Button component is powered by Store UI. However, if you try to use this on your app you will see that the button is lacking styles. To add styles, we will use CSS modules because they allow us to target data attributes. On your terminal, type:
+```sh
+touch src/components/ui/Button/Button.module.css
+```
+
+Now, on `Button.module.css`:
+```css
+[data-store-button]: {
+  @apply p-0 bg-primary-100;
+}
+```
+
+The `@apply` directive exists because we are using [Tailwind CSS](https://tailwindcss.com/). To learn more about tailwind, [see their docs](https://tailwindcss.com/docs). To know more about our best practices on using tailwind, see the [Styling Components](#%EF%B8%8F-styling-components) section.
+This `data-store-button` is a CSS data attribute selector. To know which selectors are available, check [store-ui docs](https://storeui.netlify.app/).
+
+Now, open `Button.tsx` and import this CSS with:
+```tsx
+import React from 'react'
+import { Button as UIButton } from '@vtex/store-ui'
+import type { ButtonProps } from '@vtex/store-ui'
+
+import './Button.module.css'
+
+interface Props extends ButtonProps {}
+
+function Button (props: Props) {
+  return <UIButton {...props} />
+}
+
+export default Button
+```
+
+For most components, you would stop here. However, buttons can have different variants. For instance, suppose you want to have a button component with primary and muted variants. To add variants to the component, update `Button.tsx`:
+```tsx
+import React from 'react'
+import { Button as UIButton } from '@vtex/store-ui'
+import type { ButtonProps } from '@vtex/store-ui'
+
+import './Button.module.css'
+
+interface Props extends ButtonProps {
+  variant: 'muted' | 'primary'
+}
+
+function Button ({variant, ...props}: Props) {
+  return <UIButton className={variant} {...props} />
+}
+
+export default Button
+```
+
+and then, on `Button.module.css`:
+```css
+.primary [data-store-button]: {
+  @apply p-0 bg-primary-100;
+}
+
+.muted [data-store-button]: {
+  @apply p-0 bg-muted-100;
+}
+```
+
+Now we have a styled Button component that accepts different variants!! 🎉
+
+The aforementioned guide works well for UI components. However, sections are more complex and usually don't have variants, since they usually serve a single responsibility in the page. For sections, you can use tailwind natively like:
```

Comment:
```suggestion
The aforementioned guide works well for UI components. However, sections are more complex and usually don't have variants, since they usually serve a single responsibility on the page. For sections, you can use tailwind natively like:
```